Src/Support/GoogleApis.Core/Apis/Json/NewtonsoftJsonSerializer.cs

```diff
@@ -75,10 +76,13 @@ public static NewtonsoftJsonSerializer Instance
         static NewtonsoftJsonSerializer()
         {
             // Initialize the Newtonsoft serializer.
-            JsonSerializerSettings settings = new JsonSerializerSettings();
-            settings.NullValueHandling = NullValueHandling.Ignore;
-            settings.Converters.Add(new RFC3339DateTimeConverter());
-            newtonsoftSerializer = JsonSerializer.Create(settings);
+            s_settings = new JsonSerializerSettings

```

Comment:
So if this change isn't actually in the discovery library, do we really need the test to be in there?